src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block-stars stars--0">
+      <div class="block-stars stars__star"></div>
```

Comment:
and children's must have stars__star
<img width="859" alt="Screenshot 2023-02-16 at 17 01 38" src="https://user-images.githubusercontent.com/53228259/219401246-489b90ed-fc3d-45ce-ace8-f8b581af3b43.png">